pkg/configs/advisory/v2/advisory.go

```diff
@@ -0,0 +1,123 @@
+package v2
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+
+	"github.com/samber/lo"
+	"github.com/wolfi-dev/wolfictl/pkg/versions"
+	"github.com/wolfi-dev/wolfictl/pkg/vuln"
+)
+
+type Advisory struct {
+	ID string `yaml:"id"`
+
+	// Aliases lists any known IDs of this vulnerability in databases.
+	Aliases []string `yaml:"aliases,omitempty"`
+
+	// Events is a list of timestamped events that occurred during the investigation
+	// and resolution of the vulnerability.
+	Events []Event `yaml:"events"`
+}
+
+// Latest returns the latest event in the advisory.
+func (adv Advisory) Latest() Event {
+	if len(adv.Events) == 0 {
+		return Event{}
+	}
+
+	sorted := adv.SortedEvents()
+	return sorted[len(adv.Events)-1]
+}
+
+// SortedEvents returns the events in the advisory, sorted by timestamp, from
+// oldest to newest.
+func (adv Advisory) SortedEvents() []Event {
```

Comment:
I would specifically document that we're sorting in-place - and calling this method will have side effects!

Reply:
Unless you object, I'm going to update the method to ensure the slice is copied before sorting happens so there are no side effects!

Reply:
Works for me!